app/locale/en.coffee

```diff
@@ -1675,6 +1675,8 @@ module.exports = nativeDescription: "English", englishDescription: "English", tr
     hoc_blurb3: "activity, download a new teacher lesson plan, and tell your students to log in to play!"
     hoc_button_text: "View Activity"
     no_code_yet: "Student has not written any code for this level yet."
+    code_red: "Human's Code"
```

Comment:
Let's either using the [existing i18n keys](https://github.com/codecombat/codecombat/blob/master/app/locale/en.coffee#L2316-L2317) for this or at least call this Red and Blue on the English version, too; we're trying to move away from Humans and Ogres and just calling it Red/Blue generally.

Reply:
oh, yes ! I just missed it.